Week5/app/src/main/java/com/em4n0101/mymovies/database/MoviesDatabaseDao.kt

```diff
@@ -0,0 +1,26 @@
+package com.em4n0101.mymovies.database
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+import androidx.room.Update
+import com.em4n0101.mymovies.data.Movie
+
+@Dao
+interface MoviesDatabaseDao {
+    @Insert
```

Comment:
You can implement @Insert method to store all movies at once instead of storing them one by one.
Just pass in a list of movies to the method